app/models/clinic.rb

```diff
@@ -22,6 +23,7 @@ class Clinic
 
   # Validations
   validates :name, :street_address, :city, :state, :zip, presence: true
+  validates :name, uniqueness: true
```

Comment:
Minor, and you can roll with the way you have it but you can also use: `validates_uniqueness_of :name` which is more descriptive and you don't have to pass a param too.